src/transformers/models/llama/convert_llama_weights_to_hf.py

```diff
@@ -231,10 +240,16 @@ def permute(w):
     shutil.rmtree(tmp_model_path)
 
 
-def write_tokenizer(tokenizer_path, input_tokenizer_path):
+def write_tokenizer(tokenizer_path, input_tokenizer_path, use_fast=True):
     print(f"Fetching the tokenizer from {input_tokenizer_path}.")
     # Initialize the tokenizer based on the `spm` model
-    tokenizer = LlamaTokenizer(input_tokenizer_path)
+    tokenizer_class = LlamaTokenizer
+    if use_fast:
+        try:
+            tokenizer_class = LlamaTokenizerFast
+        except ValueError:
+            pass
```

Comment:
I'd print a message here saying `use_fast` is `True` but the fast tokenizer couldn't be loaded, as the other warning will happen when CLI is first called, so might not be spotted. This is just a weak suggestion though :)